generate-stats

```diff
@@ -100,6 +102,34 @@ def main():
     # Sort tables
     all_tables = sorted(all_tables, key=lambda x: x["data"][0][1], reverse=True)
 
+    # Select unexpected messages
+    messages = cursor.execute("""\
+            SELECT retry_reason, TestRuns.url
+            FROM Failures
+            JOIN TestRuns ON Failures.run = TestRuns.id
+            WHERE TestRuns.time > ? AND
+                retry_reason IS NOT null AND
+                retry_reason IS NOT 'be robust against unstable tests' AND
+                retry_reason IS NOT 'due to failure of test harness or framework';""", (since,)).fetchall()
```

Comment:
I'd rather have a positive check here, but this may not be entirely trivial. Since this code is totally fungible, let's start with that.